frontend/src/services/product.js

```diff
@@ -14,6 +14,11 @@ function identify (userId, set, setonce) {
             ...setonce
         })
     }
+    const _hsq = window._hsq = window._hsq || []
+    _hsq.push(['identify', {
+        id: userId,
+        email: set.email
+    }])
```

Comment:
I might be missing something, but why not:

```suggestion
    if (window._hsq) {
        window._hsq.push(['identify', {
            id: userId,
            email: set.email
        }])
    }
```

Reply:
Edit, oh perhaps hubspot loads deferred, and still checks window._hsq?

Reply:
This is how all of their examples are coded. I *think* its because they recommend using deferred loading of their module, so it is possible it hasn't been loaded when you want to use `window._hsq` - I assume their code checks to see if you've created _hsq yourself and then processes what's there before swapping it for their own object.